src/scheduler/resv_info.c

```diff
@@ -534,10 +534,10 @@ query_reservations(server_info *sinfo, struct batch_status *resvs)
 			}
 		}
 		resresv_arr[idx] = resresv;
+		resresv_arr[idx + 1] = NULL;
```

Comment:
This is really up to you, but why not put this on the other side of idx++?  It gets incremented already.  Why bother being explicit about adding one?